engine/src/main/java/org/destinationsol/game/planet/TileObjBuilder.java

```diff
@@ -27,14 +27,16 @@
 import org.destinationsol.game.SolGame;
 import org.destinationsol.game.drawables.DrawableLevel;
 import org.destinationsol.game.drawables.RectSprite;
+import org.destinationsol.game.drawables.SpriteManager;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class TileObjBuilder {
     public TileObject build(SolGame game, float size, float toPlanetRelAngle, float distance, Tile tile, Planet planet) {
         float spriteSize = size * 2;
-        RectSprite sprite = new RectSprite(tile.tex, spriteSize, 0, 0, new Vector2(), DrawableLevel.GROUND, 0, 0f, SolColor.WHITE, false);
+        //HACK: For now, using SpriteManager.createSprite breaks planet formation.
```

Comment:
Were you able to figure out why?

Reply:
I still do not know why it occurs, although I will try to fix it when I can work that out. When I used ```SpriteManager.createSprite```, the sprites appeared to have broken shapes, so that you could easily go through them and planetary turrets could shoot through walls (or be inside them competely). All the decorations were misplaced somewhat, so that they could be seen to be floating just above the ground in some cases. This does not occur when using ```SpriteManager.createStaticSprite``` though, so I presume it to be an issue the the new ```AnimatedRectSprite``` class. I will continue to try to fix that, although the solution mentioned in the code was a temporary workaround that made the game properly playable again.

You can see an image of the problem below:
![planetterrainspritebug](https://user-images.githubusercontent.com/24301287/48023608-38123100-e136-11e8-85d1-60286e9a9700.JPG)